selfdrive/ui/sunnypilot/quietmode.py

```diff
@@ -0,0 +1,41 @@
+"""
+Copyright (c) 2021-, Haibin Wen, sunnypilot, and a number of other contributors.
+
+This file is part of sunnypilot and is licensed under the MIT License.
+See the LICENSE.md file in the root directory for more details.
+"""
+from openpilot.common.params import Params
+from cereal import car
+
+AudibleAlert = car.CarControl.HUDControl.AudibleAlert
+
+class QuietDriveManager:
+  def __init__(self):
+    self.current_alert: int = AudibleAlert.none  # Explicitly declare as int
+    self.param_s = Params()
+    self.quiet_drive: bool = bool(self.param_s.get_bool("QuietDrive"))
+    self._frame = 0
+
+  def load_param(self) -> None:
+    self._frame += 1
+    if self._frame == 50:
+      self.quiet_drive = bool(self.param_s.get_bool("QuietDrive"))
+
+  def is_quiet_drive_enabled(self) -> bool:
+    return bool(self.quiet_drive)
```

Comment:
If this is only used in `QuietDriveManager` only, we can just pass `self.quiet_drive` in `should_play_sound`.